### DISCUSSION TITLE
Get Error "FETCHING EXISTING VECTORS FAILED"

### DISCUSSION THREAD

**Discussion by tigertian** [truefoundry/cognita#268] [General]
I build and run the local service. I add a chinese pdf/txt file into datasource. When I add a collection with this datasource, it always show the status "FETCHING EXISTING VECTORS FAILED". How do I solve it?

**Comment by mnvsk97** [comment]
Hello @tigertian 
Apologies for the late response. There have been a lot of changes since you raised this issue so I'm curious if you are still facing this issue? If so, please provide the following
1. attach a sample file 
2. models_config.yaml
3. Are you using a local llm inference server using `ollama`? Please share the `docker compose` command that you used.
4. 
Thanks in advance.
